snuba/snuba_migrations/generic_metrics/0060_distributions_mv4.py

```diff
@@ -0,0 +1,95 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import (
+    AggregateFunction,
+    Column,
+    DateTime,
+    Nested,
+    String,
+    UInt,
+)
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    blocking = False
+    view_name = "generic_metric_distributions_aggregation_mv_v4"
+    dest_table_columns: Sequence[Column[Modifiers]] = [
```

Comment:
This doesn't contain any of the _weighted columns?

Reply:
There are no `_weighted` columns because (from testing), it doesn't actually matter what columns are defined in `dest_table_columns`. This seems to be an artifact, see the previous [distributions mat view](https://github.com/getsentry/snuba/blob/1d3a54db057fc9ee0fdc60d227143dc9e662b0e4/snuba/snuba_migrations/generic_metrics/0047_distributions_mv3.py#L16-L39) and notice that there are no `min`, `max`, `sum`, etc. aggregation columns. However, they still exist because it is defined in the `SELECT` in the mat view.

Reply:
We should remove this completely then. Otherwise it's just confusing.